.travis.yml

```diff
@@ -2,5 +2,4 @@ language: node_js
 node_js:
   - '10'
   - '8'
-  - '7'
```

Comment:
eslint refuses to install: `error eslint@5.6.0: The engine "node" is incompatible with this module. Expected version "^6.14.0 || ^8.10.0 || >=9.10.0".`

We could ignore engines, but I don't think that's better than dropping a version of Node that has been EOL since summer 2017

Reply:
it's fine not to run on EOL never-LTS versions of node imo.

Reply:
Yeah, good point on differentiating between what was LTS at one point and what was not